csi/csi.go

```diff
@@ -125,12 +132,17 @@ func NewOsdCsiServer(config *OsdCsiServerConfig) (grpcserver.Server, error) {
 		return nil, fmt.Errorf("Failed to create CSI server: %v", err)
 	}
 
+	sdkPort := config.SdkPort
+	if len(config.SdkPort) == 0 {
```

Comment:
This seems like a bug if you are seeing a config.SdkPort being set to 0. Was this required just unit tests?

Reply:
This was for CSI Sanity test or unit test IIRC.. I can configure them on the test side though. Will make a note for the followup issue